Makefile

```diff
@@ -88,22 +88,22 @@ testshell: .env .docker-build  ## | Open shell in test environment.
 	bin/test.sh --shell
 
 .PHONY: docs
-docs:  ## | Build docs.
-	bin/build-docs-locally.sh
+docs: .env .docker-build  ## | Build docs.
+	docker-compose run --rm --user ${USE_UID} linting docs
```

Comment:
I used the `linting` container here. What we should probably do is rename it to `tools` or `utils` since it's now used for building docs and linting.